docs/runbooks/CDIDataVolumeUnusualRestartCount.md

```diff
@@ -11,28 +11,22 @@ Data volumes are responsible for importing and creating a virtual machine disk o
 
 ## Diagnosis
 
-1. Obtain the name and namespace of the data volume:
+1. Find CDI pods that are unlikely to ever resolve:
```

Comment:
This sentence appears to be unrelated to the "Meaning" and "Impact". Perhaps rewrite it like this:

"Find CDI pods with more than three restarts:"

We are not supposed to use 'jq' commands downstream. Can you substitute a 'jsonpath' command?

Reply:
This runbook was using jq prior to this, should I change all references to jq?

Reply:
If you can change all the jq references, that would be a great help for downstream docs. :-) It's not urgent. You could do so in a separate PR.

Reply:
Oh sorry this runbook only had this one jq reference, I got rid of it now